<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApproximationDialog</class>
 <widget class="QDialog" name="ApproximationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="temperatureGroupBox">
       <property name="title">
        <string>Temperature</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="TcurveListWidget"/>
        </item>
        <item>
         <widget class="QCheckBox" name="TrememberCheckBox">
          <property name="text">
           <string>Remember choice</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Tapproximation0">
          <property name="text">
           <string>Line (y=ax+b)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Tapproximation1">
          <property name="text">
           <string>Exp (y=(b-a) exp (-x/c) + a)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Tapproximation2">
          <property name="text">
           <string>Exp + Line (y=(b-a)*exp (-x/c) +a +d*x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Tapproximation3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Exp + Exp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Tapproximation4">
          <property name="text">
           <string>Exp + Exp + Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="FrequencyGroupBox">
       <property name="title">
        <string>Frequency</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="FcurveListWidget"/>
        </item>
        <item>
         <widget class="QCheckBox" name="FrememberCheckBox">
          <property name="text">
           <string>Remember choice</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Fapproximation0">
          <property name="text">
           <string>Line (y=ax+b)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Fapproximation1">
          <property name="text">
           <string>Exp (y=(b-a) exp (-x/c) + a)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Fapproximation2">
          <property name="text">
           <string>Exp + Line (y=(b-a)*exp (-x/c) +a +d*x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Fapproximation3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Exp + Exp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Fapproximation4">
          <property name="text">
           <string>Exp + Exp + Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ApproximationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ApproximationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
